test/functional/ui/highlight_spec.lua

```diff
@@ -303,3 +303,248 @@ describe('Default highlight groups', function()
     ]], {[1] = {foreground = Screen.colors.Red, background = Screen.colors.Green}})
   end)
 end)
+
+describe('Cursorline highlight', function()
+  -- Test the default attributes for highlight groups shown by the :highlight
+  -- command
+  local screen
+
+  local hlgroup_colors = {
+    NonText = Screen.colors.Blue,
+    Cursorline = Screen.colors.Grey90,
+    SpecialKey = Screen.colors.Red,
+  }
+
+  before_each(function()
+    clear()
+    screen = Screen.new()
```

Comment:
Please change this to `screen = Screen.new(50, 8)` so that the screen tests don't have so much empty space.